pan3d/ui/toolbar.py

```diff
@@ -22,17 +25,25 @@ def __init__(
                 classes="mx-10",
             )
             with vuetify.VBtn(
-                click=reset,
+                click=reset_function,
                 v_show=unapplied_changes,
-                # small=True,
                 variant="tonal",
             ):
                 html.Span("Apply & Render")
                 html.Span("({{ %s }})" % da_size, v_show=da_size)
```

Comment:
f-string or maybe not here since it will add more `{{}}` which can confuse the code. Then use the explicit format method.

Reply:
Changed in [28561cb](https://github.com/Kitware/pan3d/pull/33/commits/28561cb331878bed9f5131df5a01d8307360c41e)